### ISSUE TITLE
Invalid characters in documentation

### ISSUE THREAD

**Issue by urbansky** [micronaut-projects/micronaut-core#2553]
It sounds strange, but when I copy source code from the documentation and paste it into my IDE I get error messages.

For example, when I click on 'Copy to Clipboard' at https://docs.micronaut.io/1.3.0.M1/guide/index.html#_scheduling_at_a_fixed_rate and paste, I get an error message in IntelliJ and with Gradle-Build the following error message:

'unexpected token: ​ @'

If I delete a hidden character before 'void' everythings works fine.

I know that is a strange issue. But for beginners it can be very confusing.

**Comment by sranjha** [comment]
I observed that this issue only occurs in Chrome and not in Firefox or Edge, and also not for all code snippets. After digging around for a while, it seems that the CSS rule

```
pre {
white-space: pre-wrap;
}
```

causes the zero-width spaces to appear. I have found that switching this off or adding the rule

```
pre.highlight {
    white-space: pre;
}
```

removes the invalid characters. I think this would have to be done in micronaut-docs, but I'm not sure in which stylesheet it belongs.

**Comment by utsav0209** [comment]
@graemerocher  If we  exclude `white-space: pre-wrap;` or use  `white-space: pre;` it will cause a horizontal scroll-bar to come, if that's fine then I think I know what to change

![image](https://user-images.githubusercontent.com/42451248/116795210-dfe2fc80-aaf0-11eb-9264-49067fb6e9d5.png)

**Comment by graemerocher** [comment]
@utsav0209 probably better than having broken code copied. Thanks for looking into t

**Comment by mattmoss** [comment]
Verified that this is fixed in latest documentation published (i.e. https://docs.micronaut.io/latest/guide)

Existing published documentation (prior to 2.5.13 for 2.x, prior to 3.0.1 for 3.x) still has the issue. Beginners should now be looking to use latest stuff, which works.
